bin/snyk_tests.sh

```diff
@@ -22,13 +24,39 @@ source bin/common/check_returncode.sh
 # authenticate snyk CLI locally
 run_and_check_returncode "snyk auth $SNYK_AUTHENTICATION_TOKEN"
 
-# Iac run limited to 20 runs a month. TBD Discuss with Jon Velapondi.
-# echo "Scanning Infrastructure as code with Snyk for potential vulnerabilities..."
-# snyk iac test || true
-
 # scan orca website
 echo "Scanning open-source code with Snyk for potential vulnerabilities..."
 cd website
 snyk test -d --severity-threshold=high || true
+cd -
+
+# run snyk test for shared libraries
+echo
+echo "Running snyk tests in shared_libraries"
+echo
+cd shared_libraries
+snyk test --severity-threshold=high --file=requirements.txt --package-manager=pip || true
+cd -
+
+function run_snyk_tests() {
+  echo
+  echo "Running snyk test in $1"
+  echo
+
+  cd $1
+  # remove shared library from the requirement files since snyk cannot scan through it
```

Comment:
snyk cannot scan through ../../shared_libraries[all] even after it was isntalled via pip locally.
Decided to ignore it for now to get the scan working.